generate_nibetaseries_images.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+set -e
+
+# Generate Dockerfile.
+generate_docker() {
+  docker run --rm kaczmarj/neurodocker:0.5.0 generate docker \
```

Comment:
I selected a specific version of `kaczmarj/neurodocker` (i.e. 0.5.0) versus the master release since I'm bias towards being as explicit as possible for what code is being run.

Reply:
Yeah, much better than my unspecific `master` option, sorry for missing this!